patches/react-dev-utils+10.2.1.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/react-dev-utils/webpackHotDevClient.js b/node_modules/react-dev-utils/webpackHotDevClient.js
```

Comment:
please add, JIRA backlog issue to perform an **react EJECT**

Reply:
Done.
See: https://mapcolonies.atlassian.net/jira/software/c/projects/MAPCO/boards/94?modal=detail&selectedIssue=MAPCO-2099&assignee=6012d42ff564b600711a03c3